sites/fast-creator/app/preview.tsx

```diff
@@ -231,6 +233,7 @@ class Preview extends Foundation<{}, {}, PreviewState> {
                         );
                         break;
                     case MessageSystemType.custom:
+                        console.log("ugh", messageData);
```

Comment:
*sigh*

Reply:
Just, uh, checking to see if reviewers were paying attention...